contract-map.json

```diff
@@ -230,6 +246,23 @@
     "decimals": 18,
     "symbol": "META"
   },
+  "0x9a642d6b3368ddc662CA244bAdf32cDA716005BC": {
+    "name": "Qtum",
+    "logo": "qtum_28.png",
+    "erc20": true,
+    "decimals": 18,
+    "symbol": "QTUM"
+  },
+  "0x7d4b8Cce0591C9044a22ee543533b72E976E36C3": {
+    "name": "Change Coin",
```

Comment:
I can't find anywhere they refer to their token as "Change Coin". Might be more accurate to call this "Changebank"?

Reply:
Etherscan shows as ChangeCoin

Reply:
https://etherscan.io/address/0x7d4b8cce0591c9044a22ee543533b72e976e36c3#code

Line 218 :
Token name is "Change COIN"

Reply:
There it is. Thank you.